### DISCUSSION TITLE
Order pesanan dari Web ke WA

### DISCUSSION THREAD

**Discussion by mediasaranainformatika** [kirimwa-id/forum#87] [Tanya Jawab] [answered]
Salam kak **RIO**,

Mohon maaf sebelumnya, sya tertarik dengan API yang kak RIO kembangkan. Kebetulan saya ada web jualan, rencananya ingin mengintegrasikan dengan KirimWA API. Saat customer selesai melakukan order produk, pesan akan terkirim ke WA Toko lewat API ini. Jika dimungkinkan, endpoint mana yang bisa digunakan kak? Apakah tetap menggunakan paramater ini dengan `POST/messages ?

`'token' => getenv('API_TOKEN'),` 
`      'url' => 'https://api.kirimwa.id/v1/messages',`
`      'method' => 'POST',`
`      'payload' => json_encode([`
  `         'message' => 'Halo ini adalah pesan dari api.kirimwa.id.',`
  `         'phone_number' => '6281234567890', <-- no ini no toko atau no pelanggan`
  `         'message_type' => 'text',`
  `         'device_id' => 'iphone-x-pro'`
`    ])`
`  ];
`

**Comment by rioastamal** [comment, marked as answer]
@mediasaranainformatika betul, sama dengan mengirim `POST /v1/messages` biasa, cukup dengan nomor tujuan adalah nomor WA toko karena pesan tsb ditujukan ke nomor WA dari toko.

**Reply by mediasaranainformatika** [reply]
Ok kak RIO. Sya daftar dulu KirimWA.id nya. Terima kasih kak.

**Comment by Muhhadin** [comment]
Nyinak.. (masit baru )
